Added in option to purchase merchandise

mdalmazan opened the pull request:
Fixes #1467 

Added in option to purchase merchandise. Located in the "Get Involved: Funding"

cassidyjames commented on get-involved.php:
```diff
@@ -56,14 +56,15 @@
             <p>Patreon works like an ongoing crowdfunding campaign. Choose an amount, get rewards, and help us reach our goals. <a class="read-more" href="https://www.patreon.com/" target="blank">Learn More</a></p>
             <p>PayPal is a quick and easy solution. Choose a one-time amount or set up a subscription.</p>
-
+            <p>Buying merchandise directly supports the development of Elementary, making it better for everyone.</p>
```
elementary is always lowercased, see elementary.io/brand

cassidyjames replied:
Just for completeness-sake for anyone looking at this later, on #1467 I commented that we could change this to 

> Buying merch from our store is another way to help us financially, plus you can get some exclusive elementary gear to show your support to friends, family, and coworkers.

cassidyjames commented on get-involved.php:
```diff
@@ -56,14 +56,15 @@
                 <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                 <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                 <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
+                <a class="button flat" href="https://elementary.io/store/" target="_blank">Merchandise</a>
```
I think since this is staying on our site, it can be `/store` instead of the FQDN. I think there's also something to do with PHP to make sure we keep the user's language, but I'll check to confirm that. Lastly, I think `Store` for the title makes more sense since that's what we use in the nav bar.

cassidyjames replied:
Ah yeah, for reference, in the header we have:

```php
<a href="<?php echo $page['lang-root'].'store/'; ?>">Store</a>
```

so I think 

```php
<a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>">Store</a>
```

works here.

mdalmazan replied:
Alright I can do that. I don't know much PHP so all that will just confuse me.

mdalmazan commented:
Alright I changed wording and did some PHP magic to the store link

lewisgoddard commented on get-involved.php:
```diff
@@ -56,14 +56,15 @@
                 <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                 <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
+                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
                 </form>
```
Okay, so the problem is that the form inputs aren't indented so it isn't obvious, but this form tag needs to be moved up a line.

```
                <form action="https://www.paypal.com/cgi-bin/webscr" id="paypalform" method="post" target="_top">
                <input type="hidden" name="cmd" value="_s-xclick">
                <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
                </form>
```

```
                <form action="https://www.paypal.com/cgi-bin/webscr" id="paypalform" method="post" target="_top">
                    <input type="hidden" name="cmd" value="_s-xclick">
                    <input type="hidden" name="hosted_button_id" value="LG382EHQVTDYN">
                    <input type="submit" value="PayPal" name="submit" title="PayPal - The safer, easier way to pay online!" class="button flat">
                    <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
                </form>
                <a class="button flat" href="<?php echo $page['lang-root'].'store/'; ?>" target="_blank">Store</a>
```

mdalmazan replied:
Yeah I noticed that.  I just got home so I'll make all the requested changes.

mdalmazan commented:
I was having some issues with git because git just dislikes me. I hope I've addressed your issues. Thank you for dealing with me for my first PR. It means a lot. :)

cassidyjames commented:
Hey @mdalmazan, I'm rolling this change into a larger change of the Funding section. Thanks for your code contribution, and we hope to see you contribute more in the future. :smile:
